packages/contracts/test/cyberMarketplace.spec.ts

```diff
@@ -0,0 +1,210 @@
+import chai from 'chai'
+import { solidity } from 'ethereum-waffle'
```

Comment:
where is needed?

Reply:
nowhere, should've cleanup unused imports.

Reply:
yeah check in my existing test suite file, I don't think this one is needed